app/src/main/res/layout/activity_main.xml

```diff
@@ -106,41 +106,46 @@
 
     <LinearLayout
         android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_height="100dp"
```

Comment:
Why has the `android:layout_height` been made `100dp` instead of `wrap_content` ? @Rohan-RPJ